data/json/overmap/overmap_special/alien.json

```diff
@@ -13,5 +13,96 @@
+      { "point": [ 1, 2, 2 ], "overmap": "exodii_base_x1y2z2_north" },
+      { "point": [ 2, 2, 2 ], "overmap": "exodii_base_x2y2z2_north" },
+      { "point": [ 3, 2, 2 ], "overmap": "exodii_base_x3y2z2_north" },
+      { "point": [ 0, 3, 2 ], "overmap": "exodii_base_x0y3z2_north" },
+      { "point": [ 1, 3, 2 ], "overmap": "exodii_base_x1y3z2_north" },
+      { "point": [ 2, 3, 2 ], "overmap": "exodii_base_x2y3z2_north" },
+      { "point": [ 3, 3, 2 ], "overmap": "exodii_base_x3y3z2_north" },
+      { "point": [ 0, 0, 3 ], "overmap": "exodii_base_x0y0z3_north" },
+      { "point": [ 1, 0, 3 ], "overmap": "exodii_base_x1y0z3_north" },
+      { "point": [ 2, 0, 3 ], "overmap": "exodii_base_x2y0z3_north" },
+      { "point": [ 3, 0, 3 ], "overmap": "exodii_base_x3y0z3_north" },
+      { "point": [ 0, 1, 3 ], "overmap": "exodii_base_x0y1z3_north" },
+      { "point": [ 1, 1, 3 ], "overmap": "exodii_base_x1y1z3_north" },
+      { "point": [ 2, 1, 3 ], "overmap": "exodii_base_x2y1z3_north" },
+      { "point": [ 3, 1, 3 ], "overmap": "exodii_base_x3y1z3_north" },
+      { "point": [ 0, 2, 3 ], "overmap": "exodii_base_x0y2z3_north" },
+      { "point": [ 1, 2, 3 ], "overmap": "exodii_base_x1y2z3_north" },
+      { "point": [ 2, 2, 3 ], "overmap": "exodii_base_x2y2z3_north" },
+      { "point": [ 3, 2, 3 ], "overmap": "exodii_base_x3y2z3_north" },
+      { "point": [ 0, 3, 3 ], "overmap": "exodii_base_x0y3z3_north" },
+      { "point": [ 1, 3, 3 ], "overmap": "exodii_base_x1y3z3_north" },
+      { "point": [ 2, 3, 3 ], "overmap": "exodii_base_x2y3z3_north" },
+      { "point": [ 3, 3, 3 ], "overmap": "exodii_base_x3y3z3_north" },
+      { "point": [ 0, 0, 4 ], "overmap": "exodii_base_x0y0z4_north" },
+      { "point": [ 1, 0, 4 ], "overmap": "exodii_base_x1y0z4_north" },
+      { "point": [ 2, 0, 4 ], "overmap": "exodii_base_x2y0z4_north" },
+      { "point": [ 3, 0, 4 ], "overmap": "exodii_base_x3y0z4_north" },
+      { "point": [ 0, 1, 4 ], "overmap": "exodii_base_x0y1z4_north" },
+      { "point": [ 1, 1, 4 ], "overmap": "exodii_base_x1y1z4_north" },
+      { "point": [ 2, 1, 4 ], "overmap": "exodii_base_x2y1z4_north" },
+      { "point": [ 3, 1, 4 ], "overmap": "exodii_base_x3y1z4_north" },
+      { "point": [ 0, 2, 4 ], "overmap": "exodii_base_x0y2z4_north" },
+      { "point": [ 1, 2, 4 ], "overmap": "exodii_base_x1y2z4_north" },
+      { "point": [ 2, 2, 4 ], "overmap": "exodii_base_x2y2z4_north" },
+      { "point": [ 3, 2, 4 ], "overmap": "exodii_base_x3y2z4_north" },
+      { "point": [ 0, 3, 4 ], "overmap": "exodii_base_x0y3z4_north" },
+      { "point": [ 1, 3, 4 ], "overmap": "exodii_base_x1y3z4_north" },
+      { "point": [ 2, 3, 4 ], "overmap": "exodii_base_x2y3z4_north" },
+      { "point": [ 3, 3, 4 ], "overmap": "exodii_base_x3y3z4_north" }
+    ],
+    "locations": [ "land", "swamp", "forest" ],
+    "city_distance": [ 2, 120 ],
+    "city_sizes": [ 1, 12 ],
+    "occurrences": [ 1, 5 ],
+    "flags": [ "UNIQUE" ]
```

Comment:
20% chance to spawn such an unique base on every overmap feels like way too much.

Reply:
Yes, that's from testing, thanks for spotting. It will have to be fairly high though until we have some breadcrumbs